OpenSim/Common/RegisterTypes_osimCommon.cpp

```diff
@@ -64,7 +64,7 @@ OSIMCOMMON_API void RegisterTypes_osimCommon()
 {
   try {
 
-    Object::registerType(Connector<Object>());
+    Object::registerType(Connector<Component>());
```

Comment:
We should discuss in a dev meeting the distinction between Objects, Components and ModelComponents, and then document it somewhere. If it is not necessary to limit connectors to components, then we shouldn't.